src/api-client.js

```diff
@@ -49,8 +49,8 @@ export default {
   addLab(course, formData) {
     return axios.post(`${GRADERX_API}courses/${course}/labs`, formData)
   },
-  editLab(course, lab_id, runtime, internet, test_cases) {
-    return axios.put(`${GRADERX_API}courses/${course}/labs/${lab_id}`, { name: lab_id, runtime_limit: runtime, disable_internet: !internet, test_cases })
+  editLab(course, lab_id, formData) {
```

Comment:
mixed casing naming convention :sweat_smile:  do you use snake casing or camel casing?

Reply:
This is what happens when you program in python and JS at the same time :joy: